XenonAnalyse/main.cpp

```diff
@@ -193,65 +193,64 @@ int main(int argc, char** argv)
-            {
-                println("    0x{:X},", label);
-            }
+            println("    0x{:X},", label);
+        }
 
-            println("]");
-            println("");
-        };
+        println("]");
+        println("");
+    };
 
-    std::vector<SwitchTable> switches{};
+    std::vector<SwitchTable> switches {};
 
     println("# Generated by XenonAnalyse");
 
     auto scanPattern = [&](uint32_t* pattern, size_t count, size_t type)
+    {
+        for (const auto& section : image.sections)
         {
-            for (const auto& section : image.sections)
+            if (!(section.flags & SectionFlags_Code))
             {
-                if (!(section.flags & SectionFlags_Code))
-                {
-                    continue;
-                }
+                continue;
+            }
 
-                size_t base = section.base;
-                uint8_t* data = section.data;
-                uint8_t* dataStart = section.data;
-                uint8_t* dataEnd = section.data + section.size;
-                while (data < dataEnd && data != nullptr)
+            size_t base = section.base;
+            uint8_t* data = section.data;
+            uint8_t* dataStart = section.data;
+            uint8_t* dataEnd = section.data + section.size;
+            while (data < dataEnd && data != nullptr)
+            {
+                data = (uint8_t*)SearchMask(data, pattern, count, dataEnd - data);
+
+                if (data != nullptr)
                 {
-                    data = (uint8_t*)SearchMask(data, pattern, count, dataEnd - data);
+                    SwitchTable table {};
+                    table.type = type;
+                    ScanTable((uint32_t*)data, base + (data - dataStart), table);
 
-                    if (data != nullptr)
+                    // fmt::println("{:X} ; jmptable - {}", base + (data - dataStart), table.labels.size());
+                    if (table.base != 0)
                     {
-                        SwitchTable table{};
-                        table.type = type;
-                        ScanTable((uint32_t*)data, base + (data - dataStart), table);
-
-                        // fmt::println("{:X} ; jmptable - {}", base + (data - dataStart), table.labels.size());
-                        if (table.base != 0)
-                        {
-                            ReadTable(image, table);
-                            printTable(table);
-                            switches.emplace_back(std::move(table));
-                        }
-
-                        data += 4;
+                        ReadTable(image, table);
+                        printTable(table);
+                        switches.emplace_back(std::move(table));
                     }
-                    continue;
+
+                    data += 4;
                 }
+                continue;
             }
-        };
+        }
+    };
 
-    uint32_t absoluteSwitch[] =
-    {
+    uint32_t absoluteSwitch[] = {
```

Comment:
Oversight in clang-format config https://github.com/llvm/llvm-project/issues/90837

Reply:
Set `BreakBeforeBraces` to either `Allman`, to match the original styling. It keeps all braces on the next line, even array initializers.

Reply:
This is already set, it doesn't affect array initialisers properly.